### DISCUSSION TITLE
Cannot install client on Intel i9 PC

### DISCUSSION THREAD

**Discussion by Twilight-Computer** [Ylianst/MeshCentral#6521] [General]
I have been successfully using Mesh Central for some time across a range of different client OS and versions.
Today, I cannot install the MeshAgent64 on a new Dell i9 Precision desktop Win 11 Pro
Running the install .exe generates a brief command screen (cannot read any detail) and then closes. No output / nothing logged. 
Any suggestions / debugs / tricks would be appreciated.
Server Version 1.1.33

**Comment by si458** [comment]
Windows 11 24h2 by any chance :)

Yep, the current version doesn't work unless u install the wmic feature add on before hand then install meshagent

But the is a new agent which has been build and available with steps for people to test which doesn't require wmic anymore
 https://github.com/Ylianst/MeshCentral/issues/6162#issuecomment-2461597608

Other issues about it https://github.com/Ylianst/MeshAgent/issues/254

**Reply by Twilight-Computer** [reply]
Yes Windows 11 24H2
Thanks for the quick response ! I installed wmic and it has allowed me to complete the install.

**Reply by si458** [reply]
Glad it works!
As i explained the is a new meshagent for people to try which doesnt need the wmic!
But u have to upload etc urself and if ur not comfy with it, just install the wmic for now and carry on using the current version :)

**Comment by Tomtreas** [comment]
First ran across this with a new laptop I had to deploy which was the first one I receive that already had 24H2 loaded on it.  This worked perfectly.  WIll continue to test this one on one of my workstations I frequently connect to here as well. Please update the comment when it is deemed ok to release the noAgentUpdate.  Thank you Ylianst!!

**Reply by si458** [reply]
will keep everyone informed!
but from my testing, its ok, no issues!
so you can remove the `noAgentUpdate: 1` option, then restart meshcentral and it will auto-update all your remote devices!
but again if your completely unsure, keep the option for now, and update select devices one by one and do testing 👍

**Comment by Tomtreas** [comment]
Well, I discovered that 3 old WIndows 7 64 bit systems became disconnected with the new agent.  One Win7 32 bit stayed connected and appears to have not updated as the build on it still says Dec 9 2022.  I put back the noAGentUpdate: 1 option for now and reinstalled an old agent I had saved on those client PCs.  I seem to recall seeing something about dropping support for Win7 so perhaps this is by design.  Wanted to report my findings.

**Reply by si458** [reply]
@Tomtreas the new agent was never tested with windows xp or 7, I only tested with 10 and up, so I will have to create a vm to test why it didn't work

Make sure ur Windows 7 mahcine has all the latest updates it can do

Also, yes, the agent is only 64bit,
I would have to build a 32bit version for people to test

But now a day's most people are on 64bit

**Reply by paxmobilepi** [reply]
Worked for me too, cannot believe that blocked the install of the mesh agent on my new Lenovo P16.

**Reply by si458** [reply]
glad the new agent worked!
you can actually do a work around by running `meshagent64-mygroup.exe -fullinstall` from the command line
the bug with black screen flashing and looking like its not loading and showing a `split` message is actually a GUI bug
the command line still works 👍 
the other work around is to install the WMIC add on feature in windows BEFORE you install the meshagent 👍
